auth-web/src/views/auth/home/RequestNameView.vue

```diff
@@ -37,9 +36,13 @@
       </v-col>
       <!-- Image Column -->
       <v-col cols="12" md="6">
-        <a :href="nameRequestUrl" target="_blank">
-          <v-img src="../../../assets/img/Step2_NameRequest_x2.png" aspect-ratio="1.2" contain></v-img>
-        </a>
+        <v-img
+          src="../../../assets/img/Step2_NameRequest_x2.png"
```

Comment:
Could this work with `@/assets`? Or is the assets outside the `src`?

Reply:
I don't know if Vue would do that replacement in the template. Maybe.

Hey wait a minute, didn't you write this code originally? 😁